p2p/rpc/bidderapi/v1/bidderapi.proto

```diff
@@ -296,6 +296,53 @@ message WithdrawFromWindowsResponse {
   repeated WithdrawResponse withdraw_responses = 1;
 };
 
+message PositionConstraint {
```

Comment:
Can we put this message in a proto file that both the provider and bidder protos share?